utils/dataset.py

```diff
@@ -342,47 +275,67 @@ def get_tts_datasets(path: Path,
     return train_set, val_set
 
 
-def get_binned_taco_dataloader(data_path: Path, max_batch_size: int = 8) -> BinnedTacoDataLoader:
-    train_data = unpickle_binary(data_path / 'train_dataset.pkl')
-    val_data = unpickle_binary(data_path / 'val_dataset.pkl')
+def get_binned_taco_dataloader(paths: Paths, max_batch_size: int = 8) -> BinnedTacoDataLoader:
+    train_data = unpickle_binary(paths.data / 'train_dataset.pkl')
```

Comment:
These values for filenames are hardcoded in multiple places. How about to create entries for them as well in the Paths ?

Reply:
Good idea, moved to paths.